src/ui/components/shared/LaunchBrowserModal.tsx

```diff
@@ -20,23 +20,18 @@ function LaunchBrowser({
   }, [path]);
 
   return (
-    <section className="relative m-auto w-full max-w-xl overflow-hidden rounded-lg bg-modalBgcolor text-sm text-bodyColor shadow-lg">
+    <section className="relative m-auto w-full max-w-xl overflow-hidden rounded-lg border border-inputBorder bg-modalBgcolor text-sm text-bodyColor shadow-lg">
       <div className="flex flex-col items-center space-y-9 p-6">
         <div className="place-content-center space-y-3">
           <img className="mx-auto h-12 w-12" src="/images/logo.svg" />
         </div>
         <div className="flex flex-col items-center space-y-4 text-center">
-          <div className="text-lg font-bold">Launching Replay ...</div>
-          <div className="space-y-6">
-            <p>
-              Click <strong>Open Replay</strong> in the dialog shown by your browser
-            </p>
-            {children}
-          </div>
+          <div className="text-lg font-bold">Launching Replay...</div>
+          <div className="space-y-6">{children}</div>
           <div className="w-full" />
-          <div className="flex flex-row text-xs text-gray-500">
+          <div className="flex flex-row text-xs text-gray-400">
```

Comment:
gray-400 doesn't really work for light mode

<img width="463" alt="Screen Shot 2023-08-14 at 2 27 53 PM" src="https://github.com/replayio/devtools/assets/29597/b87268e5-3fa0-455f-bef2-8dd663065b90">

<img width="435" alt="Screen Shot 2023-08-14 at 2 28 00 PM" src="https://github.com/replayio/devtools/assets/29597/85cde35b-b57a-4650-a6a6-d3be8389f7f9">


I think this is why we should be using CSS vars for stuff like this

Reply:
Absolutely. 99% of the time I'm using vars and moving things into css modules.

This particular case is an example of light and dark being different by design. Light theme doesn't need (and shouldn't have) a border and dark theme very much needs one. So I could get away with the tailwind variable in this case to get exactly the look I want.

Reply:
> Light theme doesn't need (and shouldn't have) a border and dark theme very much needs one

I don't really understand. They both look equally low contrast to me without the border :shru

Reply:
It's fine to land it if you think this looks right

Reply:
Ah, right. The difference is drop shadows. Ideally borders wouldn't be necessary at all, and in light theme you can use a drop shadow to distinguish between areas. Dark theme can't do that unless you use an electric blue or pink or something. So you end up with borders.